todo_app/app.py

```diff
@@ -1,41 +1,77 @@
 from flask import Flask, render_template, request
+from flask.globals import session
 from werkzeug.utils import redirect
-from todo_app.data.session_items import (
-    add_item,
-    complete_item,
-    delete_item,
-    get_items,
-    uncomplete_item,
-)
+from todo_app.data.items_backend import Item
+from todo_app.data.session_items import SessionItems
 
+from todo_app.data.trello_items import TrelloItems
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
+SESSION_BACKEND = "Session"
+TRELLO_BACKEND = "Trello"
+
+
+def clear_session_data():
+    keys = list(session.keys())
+    for key in keys:
+        session.pop(key)
+
+
+def get_item_backend_name():
+    return session.get("ItemBackend") or SESSION_BACKEND
+
+
+def set_item_backend(to: str):
+    session["ItemBackend"] = to
+
+
+def get_item_backend():
+    specified_backend = get_item_backend_name()
+    if specified_backend == SESSION_BACKEND:
+        return SessionItems.load()
```

Comment:
Is there any particular advantage to having this `load` method, rather than just having an empty constructor that returns the same thing?